pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -32,8 +32,6 @@ const (
 
 // groups
 const (
-	UnauthenticatedUsername = "system:anonymous"
```

Comment:
@deads2k do you want these changes in #16517?

Reply:
> @deads2k do you want these changes in #16517?

don't care